typescript-timers/interval.ts

```diff
@@ -0,0 +1,17 @@
+const $message = document.querySelector('h1');
+
+function countDown(): void {
+  if (!$message) throw new Error('$message not found!');
+  if (!($message.textContent === null)) {
+    if ($message.textContent.match(/\d+/)) {
+      if ($message.textContent === '0') {
```

Comment:
You should use a variable rather than the value of a DOM element. With a variable we have direct access to a value and can modify it easily and then we just update the content on the page.

By using the `textContent` of an element you're jumping through a lot more hoops to run the action.